build.gradle

```diff
@@ -36,15 +36,51 @@ buildscript {
 
 repositories {
     mavenCentral()
+    maven {
+        // Confluent repo for kafka-avro-serializer
+        url "https://packages.confluent.io/maven/"
+    }
 }
 
 dependencies {
-    compile 'org.apache.kafka:kafka-clients:2.4.1'
-    compile 'com.github.luben:zstd-jni:1.4.3-1'
-    compile 'org.slf4j:slf4j-api:1.7.28'
-    compile 'org.lz4:lz4-java:1.6.0'
+    compile('io.confluent:kafka-avro-serializer:5.5.1') {
+        exclude group: 'org.apache.kafka', module:'kafka-clients'
+    }
+    compile('io.confluent:kafka-schema-serializer:5.5.1') {
+        exclude group: 'org.apache.kafka', module:'kafka-clients'
+    }
+    compile 'io.confluent:common-config:5.5.1'
+    compile 'org.apache.avro:avro:1.9.2'
+    compile('io.confluent:kafka-schema-registry-client:5.5.1') { // not open source!
```

Comment:
🤔 I'm finding Apache-licensed source here: https://github.com/confluentinc/schema-registry/blob/5.5.x/client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClient.java

Reply:
Right! was a pinpoint I forgot to remove